torch/nn/quantized/modules/linear.py

```diff
@@ -113,11 +120,17 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
         self.zero_point = int(state_dict[prefix + 'zero_point'])
         state_dict.pop(prefix + 'zero_point')
 
+        self.dtype = state_dict[prefix + 'dtype']
+        state_dict.pop(prefix + 'dtype')
+
         super(Linear, self)._load_from_state_dict(state_dict, prefix, local_metadata, False,
                                                   missing_keys, unexpected_keys, error_msgs)
 
     @torch.jit.export
     def __setstate__(self, state):
+        # This needs to go before self.set_weight_bias
```

Comment:
Any reason?

Reply:
Yes, otherwise `self.dtype` will be uninitialize and `set_weight_bias` cannot check that value

Reply:
Might be good to add that in the comments.